ENIGMAsystem/SHELL/Universal_System/Resources/loading.cpp

```diff
@@ -134,11 +139,6 @@ namespace enigma
     // resize and center window
     enigma_user::window_set_size(windowWidth, windowHeight);
```

Comment:
What happens if you attempt to set the window size after going fullscreen?

Reply:
None apparently because we use fake fullscreen and this is just setting the window size to what it already is.

Reply:
"Apparently" sounds like an assumption and isn't good enough if you haven't tested it. I mean both platforms. I'm going to test this pr myself and will see if it will mess up what I believe it will, but I could be wrong.

Reply:
I did, I don't see any fullscreen issues on a brand new game.

Reply:
That's awesome and thank you, I was a little concerned at first but I take your word for it.

Reply:
@RobertBColton have you tried switching out of fullscreen if you started in fullscreen?